web-api/openapi/schemas/time.yml

```diff
@@ -7,14 +7,30 @@ PublicationDate:
   maxLength: 40
   example: "2000-05-14T20:22:50Z"
   description: |
-    A RFC3339 compatible date-time
+    A RFC3339 compatible date-time:
+    - can be in the future
```

Comment:
If we remove the filter maybe we should not accept documents with a publication data in the future anymore? (with some tolerances around clock drive;  Or at least prevent them from being set as candidate or similar.)

Because it would mean you can get recommendations for a scheduled to be released but not yet released document.

Reply:
good question, i'll raise it in the daily